app-modules/task/src/Filament/RelationManagers/BaseTaskRelationManager.php

```diff
@@ -149,6 +149,7 @@ public function table(Table $table): Table
             ])
             ->headerActions([
                 CreateAction::make()
+                    ->authorize('create', $this->getOwnerRecord())
```

Comment:
Does `getOwnerRecord` reference the correct Model here?

Reply:
Yep ! It's referring correct model here.